CocoaLumberjack.podspec

```diff
@@ -24,22 +24,16 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target    = '9.0'
 
-  s.default_subspecs = 'Default', 'Extensions'
+  s.module_map = 'Framework/Lumberjack/CocoaLumberjack.modulemap'
+  s.default_subspecs = 'Default'
```

Comment:
Thank you for your comment.

**s.module_map**

If you do not have a plan to support backward compatibility, 
`s.module_map` is not needed anymore.
In this case, backward compatibility means the followings.

When user uses `DDDispatchQueueLogFormatter` class
 [CocoaLumberjack 2.3]
import CocoaLumberjack.DDDispatchQueueLogFormatter
[CocoaLumberjack latest]
import CocoaLumberjack

To be honest, I also think cusutom modulemap is not needed anymore as you say😁
Without custom modulemap, CocoaPods automatically makes a normal modulemap file.
I made a PR to discuss about it.

**s.default_subspecs**
On the other hand, `s.default_subspecs` should be needed.
Because if this setting is removed from podspec, Swift Standard Libraries are always included even when user wants to use only Objective-C code.

To sum it up, I'll be gladly to decline(close) this PR if you do not have a plan to support backward compatibility about custom modulemap.

BTW, if you do not need custom modulemap, I think probably you had better change the modulemap of Lumberjack.xcodeproj a little. Each target still uses custom modulemap file that is not different from a modulemap file CocoaPods automatically makes.
Can I make an another PR to do this?